packages/titanium-docgen/generators/jsduck_generator.js

```diff
@@ -46,10 +46,10 @@ function convertAPIToLink(apiName) {
  * @return {string} markdown text with jsduck style links
  */
 function convertLinks(text) {
-	var matches = text.match(common.REGEXP_HREF_LINKS),
-		tokens,
-		replace,
-		link;
+	var matches = text.match(common.REGEXP_HREF_LINKS);
+	var tokens;
```

Comment:
same here, use `let`/`const` and assign/create the variables as needed rather than doing the old `var` hoisting at top thing.